src/skills-builder/skills-builder-header/SkillsBuilderHeader.jsx

```diff
@@ -1,11 +1,22 @@
 import React from 'react';
 import { useIntl } from '@edx/frontend-platform/i18n';
+import { breakpoints, useMediaQuery } from '@edx/paragon';
 import edXLogo from '../images/edX-logo.svg';
 import messages from './messages';
 
 const SkillsBuilderHeader = () => {
   const { formatMessage } = useIntl();
+  const isMedium = useMediaQuery({ maxWidth: breakpoints.medium.maxWidth });
```

Comment:
Seeing as we are doing this in the parent component as well (`SkillsBuilderModal`), what do you think about passing `isMedium` down as a prop to `SkillsBuilderHeader`? The value already exists in `SkillsBuilderModal` so this would eliminate the need to import the hook in two places.

Reply:
Good catch!